api/scpca_portal/test/utils/test_helpers.py

```diff
@@ -235,3 +235,9 @@ def test_multiple_occurences(self):
         expected_folder_count = 3
         actual_folder_count = str(new_path).count(new_value)
         self.assertEqual(actual_folder_count, expected_folder_count)
+
+
+class TestConvertBytesToGB(TestCase):
+    def test_conversion_successful(self):
+        size_in_bytes = 1234567890
+        self.assertEqual(utils.convert_bytes_to_gb(size_in_bytes), 1.23456789)
```

Comment:
I was going to suggest removing this but we will probably need it when we create the email template for datasets.